### PR TITLE
Add Archer2 OpenFOAM v6 of July 2018

### PR DESCRIPTION
**Pull request by kevinstratford**
Arising from user request.

### CODE REVIEW

File: apps/OpenFOAM/ARCHER2-OpenFOAM-v6.20180710/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# OpenFOAM v6
+
+This is OpenFOAM version 6
+See https://openfoam.org/download/6-linux/
+
+This directory contains a subdirectory `site` with the
+following scripts which allow one to install and compile OpenFOAM
+using the Gnu compiler collection on Archer2.
```
Comment (aturner-epcc) [inline apps/OpenFOAM/ARCHER2-OpenFOAM-v6.20180710/README.md:8]: Can you make sure ARCHER2 has the correct capitalisation throughout? Sounds petty, but as the service provider we need to get it right

**Review by aturner-epcc — CHANGES_REQUESTED** [review]
Fix ARCHER2  capitalisation

File: apps/OpenFOAM/ARCHER2-OpenFOAM-v6.20180710/README.md (hunk 1 above)
Comment (kevinstratford) [reply apps/OpenFOAM/ARCHER2-OpenFOAM-v6.20180710/README.md]: Not doing very well on this... need a script!

**Review by aturner-epcc — APPROVED** [review]
@kevinstratford I am going to approve this to get the useful info in the docs but it also suffers from the "Archer2" vs "ARCHER2" issue. If you can fix this in a future PR, that would be great.

**Pull request merged**